### PR TITLE
pythonPackages.rcfile: init at 0.1.4

### PR DESCRIPTION
**Pull request by marenz2569**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by risicle** [comment]
You may as well set `doCheck = false;` and not bother with the `nose` dependency as the single test module in the project is not included in the pypi tarball. Grabbing the source from github instead is made painful by the releases not being tagged in the repo. Also, the tests don't actually _use_ `nose`, that check in `setup.py` must be historic...

File: pkgs/development/python-modules/rcfile/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchPypi, buildPythonPackage
+, pythonOlder, isPy3k
+, nose }:
+
+buildPythonPackage rec {
+  pname = "rcfile";
+  version = "0.1.4";
+
+  disabled = pythonOlder "2.6" || (isPy3k && pythonOlder "3.3");
```
Comment (jonringer) [inline pkgs/development/python-modules/rcfile/default.nix]: we only carry python2.7,3.5,3.6,3.7,3.8 and only build the (full) package sets for 2.7 and 3.7. This check should be unnecessary

**Review by doronbehar — APPROVED** [review]

File: pkgs/development/python-modules/rcfile/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+buildPythonPackage rec {
+  pname = "rcfile";
+  version = "0.1.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ycbbzx3lshdjjin1lnypknhk7iy2vhr1z7g31y2g9l23dzapg91";
+  };
+
+  doCheck = false;
```
Comment (FRidh) [inline pkgs/development/python-modules/rcfile/default.nix]: Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

File: pkgs/development/python-modules/rcfile/default.nix [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+  meta = with stdenv.lib; {
+    description = "Configuration file loader for Python projects";
+    homepage = https://github.com/aequitas/rcfile;
```
Comment (jonringer) [inline pkgs/development/python-modules/rcfile/default.nix]:
```suggestion
    homepage = "https://github.com/aequitas/rcfile";
```

**Comment by jonringer** [comment]
I'm closing this PR due to lack of activity, if the PR author would like to pursue this matter again, please re-open the PR and address the concerns.

**Pull request closed without merging**